### ISSUE TITLE
A11y correction

### ISSUE THREAD

**Issue by vitorgomateus** [uchicago-library/research_data#40]
SiteImprove has flagged an element of navigation because it has a `role="navigation"` on the `<ul>` element. But [aria says](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/navigation_role) that this role should be in an element that wraps around a list. This issue has been found on both navigation lists on the footer and the one in the header. We are also missing `aria-label` on all navigation elements.

- [] Wrap all navigation lists with a `<nav>` or an element with `role="navigation"`.
- [] Remove the role attribute from any list.
- [] Add `aria-label` attributes to all navigation elements.
